yarn-project/p2p/src/testbench/testbench.test.ts

```diff
@@ -0,0 +1,140 @@
+import { createLogger } from '@aztec/foundation/log';
+import { sleep } from '@aztec/foundation/sleep';
+
+import { type ChildProcess, fork } from 'child_process';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import { mockTx } from '../../../circuit-types/src/mocks.js';
+import { type P2PConfig, getP2PDefaultConfig } from '../config.js';
+import { generatePeerIdPrivateKeys } from '../test-helpers/generate-peer-id-private-keys.js';
+import { getPorts } from '../test-helpers/get-ports.js';
+import { makeEnrs } from '../test-helpers/make-enrs.js';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const workerPath = path.join(__dirname, '../../dest/testbench/p2p_client_testbench_worker.js');
+const logger = createLogger('testbench');
+
+describe.skip('Gossipsub', () => {
```

Comment:
Is this skip on purpose?

Reply:
yep, theres no need to run this in ci at the moment as its just a scaffold